app/javascript/src/components/Dashboard/Contacts/Edit.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Pane/Form";
+
+const EditNotePane = ({ fetchNotes, showPane, setShowPane, note }) => {
```

Comment:
Why is `EditNotePane` kept under `Dashboard/Contacts/Edit.jsx`?

Reply:
Sorry, it's pretty redundant. I used the same format from NotePane for "Add new note". I'll remove it before the next PR.